Kitodo/src/main/java/de/sub/goobi/forms/SearchForm.java

```diff
@@ -169,7 +169,8 @@ protected void initUserList() {
         try {
             this.user.addAll(serviceManager.getUserService().getAllActiveUsersSortedByNameAndSurname());
         } catch (RuntimeException rte) {
-            logger.warn("Catched RuntimeException. List of users could be empty!");
+            Helper.setErrorMessage("errorLoadingMany", new String[] {Helper.getTranslation("aktiveBenutzer") });
+            logger.warn("RuntimeException caught. List of users could be empty!");
```

Comment:
Same here. Please first log the error message / reason and then display it to the user.